www/front_src/src/Resources/Graph/Performance/AnomalyDetection/AnomalyDetectionSlider.tsx

```diff
@@ -290,19 +294,24 @@ const AnomalyDetectionSlider = ({
             </div>
           </IconButton>
         </div>
-        <FormControlLabel
-          control={
-            <Checkbox
-              checked={isDefaultValue}
-              onChange={handleChangeCheckBox}
-            />
-          }
-          label={t(labelUseDefaultValue)}
-        />
+        <IconButton
+          disabled={isDefaultValue}
+          size="small"
+          onClick={setToDefaultValue}
+        >
+          <ResetIcon />
+          <Typography>{t(labelUseDefaultValue)}</Typography>
+        </IconButton>
```

Comment:
First use the Button component then use the startIcon prop: https://mui.com/material-ui/api/button/